libs/components/packages/src/schematics/ag-grid-migrate/ag-grid-migrate.schematic.spec.ts

```diff
@@ -15,6 +19,10 @@ interface TestSetup {
   schematic: (options: Schema) => Rule;
 }
 
+const UPDATE_TO_VERSION = fs.readJsonSync(
+  joinPathFragments(workspaceRoot, 'package.json'),
+).dependencies['ag-grid-community'];
+
```

Comment:
Reducing the places where the version is hard-coded.

Reply:
For unit tests, there's some benefit to having the version hard-coded since it'd be preferrable for a test to fail if the version is changed in another file (otherwise, it's possible to change the version and no tests would alert you to that fact). A middle ground would be to assign the version statically at the top of the file and use that variable throughout your spec.
```
const UPDATE_TO_VERSION = '31.2.0';
```

Reply:
Fair. I switched to a single test that verifies the version tested is the same as the dependency.